package.json

```diff
@@ -32,7 +32,7 @@
     "figc": "~0.0.3"
   },
   "scripts": {
-    "test": "./node_modules/mocha/bin/mocha -R spec test/*-test.js && ./node_modules/mocha/bin/mocha -R spec test/*-test.js --client.bigint=true",
```

Comment:
If mocha isn't globally installed, mocha command can't be used. Please don't change "./node_modules/mocha/bin/mocha"